config/engines.yml

```diff
@@ -141,6 +141,14 @@ phpmd:
     - "**.php"
     - "**.module"
     - "**.inc"
+radon:
+  image: codeclimate/codeclimate-radon
+  description: Python tool used to compute Cyclomatic Complexity.
+  community: true
+  enable_regexps:
```

Comment:
It's not urgent, and I saw your earlier change for `brakeman` too late to discuss it there. I just think we should figure out what the correct way to be consistent with these engines is: if we plan to auto-enable community engines at some point, we should probably include both `enable_regexps` & `upgrade_languages`. If we don't plan to auto-enable `community` engines, we should probably include neither.